themes/engineering/lib/js/engineering-stanford-news-override.behavior.js

```diff
@@ -0,0 +1,50 @@
+(function ($, Drupal) {
+
+  Drupal.behaviors.engineeringTheme = {
```

Comment:
do you think perhaps this should be something more specific? e.g, `engineeringSocialButtons` or something?  I'm thinking this name might cause conflicts down the road....

Reply:
Yeah, that should be renamed.